app/assets/stylesheets/browserslist

```diff
@@ -0,0 +1,4 @@
+last 2 versions
```

Comment:
This file was created here by foundation:install script, I dont' know why it should be here.

Reply:
Please ask #dev-frontent in Slack

Reply:
ok

Reply:
I think you can delete this file, because in root directory already exists `.browserlistrc` file